### ISSUE TITLE
Kelompok 6

### ISSUE THREAD

**Issue by dudung** [dudung/fi4002-01-2022-1#28]
# Kelompok 6
+ Judul:

No | NIM | Nama | GitHub
:-:  | :-: | :-: |  :-:
1 |  |  | 
2 |  |  | 
3 |  |  | 
4 |  |  | 
5 |  |  | 
6 |  |  | 
7 |  |  | 
8 |  |  | 

## Tugas 16-nov-2022
+ Lengkapi tabel di atas dengan mengisikan NIM, Nama dan user GitHub (disertai tautannya).
+ Selanjutnya diskusi mengenai perkembangan RBL dituliskan di issue ini sehingga dapat terpantau aktivitasnya.

**Comment by AslaRoudhatuRohmah** [comment]
Judul: Heat Transfer on Fryingpan

**Comment by dudung** [comment]
@AslaRoudhatuRohmah, Terima kasih.

**Comment by Aniesah** [comment]
> # Kelompok 6

> No	NIM			Nama							GitHub
> 1		10218023	Amanda Lathifah					https://github.com/amanda248la/ 
> 2		10219013	Huwaida Nur Asysyifa Mufarrida	http://github.com/huwaidanur
> 3		10219025	Asla Roudhatu Rohmah			http://github.com/AslaRoudhatuRohmah
> 4		10219033	Aniesah Akhyar					http://github.com/Aniesah

**Comment by amanda248la** [comment]
Nama: Amanda Lathifah
NIM: 10218023

**Comment by AslaRoudhatuRohmah** [comment]
https://drive.google.com/drive/u/0/mobile/folders/1RZDPO5qO1Hb_nQFW0B1T7Czf6juSIOd4?usp=sharing 
Buat referensi ini, ada yg udah cobain belum? Aku gabisa run soalnya ada problem sama 'str' tp ga ngerti str tuh apa

**Comment by dudung** [comment]
> https://drive.google.com/drive/u/0/mobile/folders/1RZDPO5qO1Hb_nQFW0B1T7Czf6juSIOd4?usp=sharing Buat referensi ini, ada yg udah cobain belum? Aku gabisa run soalnya ada problem sama 'str' tp ga ngerti str tuh apa

Apakah [Python str() function](https://www.w3schools.com/python/ref_func_str.asp), @AslaRoudhatuRohmah?

Bisa pesan kesalahannya di-screenshot dan disisipkan di percakapan ini? Terima kasih.

**Comment by amanda248la** [comment]
Dari yang saya dapat, errornya seperti ini pak @dudung 
![image](https://user-images.githubusercontent.com/98004896/202075140-45ac6757-0fe0-428e-9e40-119574652f66.png)

**Comment by AslaRoudhatuRohmah** [comment]
Iya, Pak @dudung. Saya juga sama seperti kak @amanda248la errornya

**Comment by dudung** [comment]
@AslaRoudhatuRohmah, bisa
1. Coba dengan pustaka celluloid seperti di https://stackoverflow.com/a/60035767/9475509
2.  Tetap dengan Matplotlib, tetapi instal dulu FFmpeg untuk Windows seperti di https://stackoverflow.com/a/14574894/9475509

Kabari bagaimana hasilnya.
